### PR TITLE
Dispose scoped instantiation service in comments

### PR DESCRIPTION
**Pull request by alexr00**

### CODE REVIEW

File: src/vs/workbench/contrib/comments/browser/commentThreadZoneWidget.ts [hunk 1]
```diff
@@ -139,9 +139,9 @@ export class ReviewZoneWidget extends ZoneWidget implements ICommentThreadWidget
 		super(editor, { keepEditorSelection: true, isAccessible: true });
 		this._contextKeyService = contextKeyService.createScoped(this.domNode);
 
-		this._scopedInstantiationService = instantiationService.createChild(new ServiceCollection(
+		this._scopedInstantiationService = this._globalToDispose.add(instantiationService.createChild(new ServiceCollection(
```
Comment (benibenj) [inline src/vs/workbench/contrib/comments/browser/commentThreadZoneWidget.ts:142]: Adding comment to verify https://github.com/microsoft/vscode-pull-request-github/issues/5924

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

Comment (benibenj) [inline src/vs/workbench/contrib/comments/browser/commentThreadZoneWidget.ts:142]: https://github.com/microsoft/vscode/blob/a141edab91e53a845e0a4e193dd783704b52634a/src/vs/workbench/electron-sandbox/actions/developerActions22.ts

Comment (benibenj) [inline src/vs/workbench/contrib/comments/browser/commentThreadZoneWidget.ts:142]: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

Comment (benibenj) [inline src/vs/workbench/contrib/comments/browser/commentThreadZoneWidget.ts:142]: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

**Review by benibenj — COMMENTED** [review]
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

**Review by bpasero — APPROVED** [review]

**Pull request merged**